packages/daimo-api/src/contract/paymaster.ts

```diff
@@ -136,10 +121,11 @@ export class Paymaster {
     // Sign paymaster for any valid Daimo account, excluding name blacklist.
     // Everyone else gets the Pimlico USDC paymaster.
     const isSponsored =
+      !chainConfig.chainL2.testnet &&
```

Comment:
nice, with this change I think we can also add nice rate limits on paymaster spending per user / per unit time and enforce beneficiary = our address that would cleanly solve problems around paymaster griefing -- doesn't have to be this PR I guess